docs/chapters/02_getting-started.md

```diff
@@ -153,13 +155,13 @@ To do so, we have two possible solutions:
 - Add the region to the profile of your choice in your `~/.aws/credentials` file
 
 ```ini
- ~/.aws/credentials
+  ~/.aws/credentials
 [default]
 aws_access_key_id = <DEFAULT ACCESS KEY ID>
 aws_secret_access_key = <DEFAULT SECRET ACCESS KEY>
 region=<DEFAULT REGION>
 
-[other_profile]  Give this profile the name that works best for you
+[other_profile]   Give this profile the name that works best for you
```

Comment:
why do we have this spaces at the beginning of the sentences? " Give this profile..." in place of "Give this profile...". This repeats in many parts of the files